webapp/vite.config.ts

```diff
@@ -0,0 +1,39 @@
+// Plugins
+import vue from '@vitejs/plugin-vue'
+import vuetify, { transformAssetUrls } from 'vite-plugin-vuetify'
+import VueI18nPlugin from '@intlify/unplugin-vue-i18n/vite';
+
+// Utilities
+import { defineConfig } from 'vite'
+import { fileURLToPath, URL } from 'node:url'
+import path from 'path'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue({
+    template: { transformAssetUrls }
+  }),
+  // https://github.com/vuetifyjs/vuetify-loader/tree/next/packages/vite-plugin
+  vuetify({
+    autoImport: true,
+  }),
+  VueI18nPlugin({
+    include: [path.resolve(__dirname, './src/locale/*.json')],
+  })],
+  define: { 'process.env': {} },
+  resolve: {
+    alias: {
+      '@': fileURLToPath(new URL('./src', import.meta.url))
+    },
+    extensions: ['.js',
+      '.json',
+      '.jsx',
+      '.mjs',
+      '.ts',
+      '.tsx',
+      '.vue',],
+  },
+  server: {
+    port: 3000,
+  },
```

Comment:
I assume this is the default ? Why do we specify it ? Is this for docker setup ?

Reply:
Yes, it's the default config. I haven't worked on a Docker setup for it; I thought we were serving the generated pages through Django.

Reply:
> I thought we were serving the generated pages through Django.

No, the compiled multi pages apps will be served by the legacy app / nginx.

> Yes, it's the default config. I haven't worked on a Docker setup for it;

According to https://vitejs.dev/config/server-options.html#server-port this is not the default, please remove this. The only thing that might need changing is the listen address, so we can reach the dev server within a docker container.